src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -48,6 +54,14 @@ public static void generateReposReport(List<RepoConfiguration> configs, String o
                 continue;
             }
 
+            Path configJsonPath = Paths.get(
+                    config.getRepoRoot(), REPOSENSE_CONFIG_FOLDER, REPOSENSE_CONFIG_FILE).toAbsolutePath();
+
+            if (Files.exists(configJsonPath)) {
+                StandaloneConfig standaloneConfig =  new StandaloneConfigJsonParser().parse(configJsonPath);
```

Comment:
Can we have a `try catch` here, so that if we fail to parse, so that if the `json` file is somewhat corrupted or malformed, we can skip it and stick with the original config?